WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -97,10 +108,19 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 			}
 		}
 
+		private bool IsWalletSelectable(Wallet wallet)
+		{
+			if (ManuallyChoosenWallets.TryGetValue(wallet.WalletName, out var enabled))
+			{
+				return enabled;
```

Comment:
is it the intention to also have a non-persistent way of disabling auto-coinjoin? it seems like the right behavior but maybe this should have a unit test, as this is quite different from the other sort of obvious way of implementing (as a logical OR).

Reply:
> is it the intention to also have a non-persistent way of disabling auto-coinjoin?

Yes, it is. There are two possible situations: 

* Wallet AutoCoinjoin is `false` but the user is alerted and want to boost the wallet privacy anyway and,
* Wallet `AutoCoinjoin` is `true` but the user wants to stop it. This is what I called "pause" because it can be started again using the `AddWallet` and it will start again automatically next time we start the software because the option is not persisted anywhere.